voltixApp/VoltixApp/Services/BalanceService.swift

```diff
@@ -10,33 +10,27 @@ import Foundation
 class BalanceService {
     
     static let shared = BalanceService()
-    
     private let utxo = BlockchairService.shared
     private let thor = ThorchainService.shared
     private let sol = SolanaService.shared
     private let gaia = GaiaService.shared
     
     func balance(for coin: Coin) async throws -> (coinBalance: String, balanceUSD: String) {
-        await CryptoPriceService.shared.fetchCryptoPrices()
         
         switch coin.chain {
         case .bitcoin, .bitcoinCash, .litecoin, .dogecoin:
             let blockChairData = try await utxo.fetchBlockchairData(address: coin.address,coin: coin)
-            let balanceUSD = blockChairData?.address?.balanceInUSD ?? "US$ 0,00"
-            let coinBalance = blockChairData?.address?.balanceInBTC ?? "0.0"
-            coin.rawBalance = String(blockChairData?.address?.balance ?? 0)
-            coin.priceRate = await CryptoPriceService.shared.cryptoPrices?.prices[coin.priceProviderId]?["usd"] ?? 0.0
+            coin.rawBalance = blockChairData?.address?.balance?.description ?? "0"
+            coin.priceRate = await CryptoPriceService.shared.getPrice(priceProviderId: coin.priceProviderId)
+            let balanceUSD = coin.balanceInUsd
```

Comment:
can we please rename the variable to avoid saying `balanceUSD` , because it might not be USD , the name becomes confusing

Reply:
I've renamed everything that had usd to Fiat.